airflow/executors/dask_executor.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import distributed
+
+import subprocess
+
+from airflow import configuration
+from airflow.executors.base_executor import BaseExecutor
+from airflow.utils.state import State
+
+
+class DaskExecutor(BaseExecutor):
+    """
+    DaskExecutor submits tasks to a Dask Distributed cluster.
+    """
+    def __init__(self, cluster_address=None):
+        if cluster_address is None:
+            cluster_address = configuration.get('dask', 'cluster_address')
+        if not cluster_address:
+            raise ValueError(
+                'Please provide a Dask cluster address in airflow.cfg')
+        self.cluster_address = cluster_address
+        super(DaskExecutor, self).__init__(parallelism=0)
+
+    def start(self):
+        self.client = distributed.Client(self.cluster_address)
+        self.futures = {}
+
+    def execute_async(self, key, command, queue=None):
```

Comment:
Does Dask not support queues? Than this should be noted in the documentation and it should be logged as a warning if the queue is anything different from the default.